tokio-console/CHANGELOG.md

```diff
@@ -3,6 +3,29 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.2.0 - (2024-06-07)
```

Comment:
Do you think it's a bug in release-plz? 👀 
For the issue of checks not being run, check our [these](https://release-plz.ieni.dev/docs/github/token) docs 👍

Reply:
Thanks for the link to the docs! I think I would have saved myself some time if I'd checked them earlier (I worked this out by myself yesterday, but it took a bit of hunting).

Regarding the version, I don't think it's a bug, but when creating a commit that's marked as a breaking change, I would like a way to specify for **which** crates it's a breaking change, as in this case it may be for some crates in the workspace, but not others, even though the other crates have code changes in that commit. I don't think that's currently possible, is it?

Reply:
Yes. You are right.
If you have nice ideas about this, feel free to open an issue 👍